src/app/helpers/jwt.interceptor.spec.ts

```diff
@@ -24,24 +34,37 @@ describe('JWTInterceptor', () => {
     });
   });
 
+  const setUp = (authService: AuthService) => {
+    service = new JwtInterceptor(authService)
+  };
+
   it('should create', () => {
-    expect(jwtInterceptor).toBeTruthy();
+    setUp(authServiceData);
+    expect(service).toBeTruthy();
   });
 
   it('should auto populate jwt headers in the request', () => {
+    setUp(authServiceData);
     // arrange
     httpHandlerSpy.handle.and.returnValue(of({
       data: {
         message: 'data'
       }
     }));
     // act
-    jwtInterceptor.intercept(httpRequestSpy, httpHandlerSpy)
+    service.intercept(httpRequestSpy, httpHandlerSpy)
       .subscribe(
         result => {
```

Comment:
this result is declared but the value is never read

Reply:
Some changes have come up due to rebasing.